src/app/services/api.ts

```diff
@@ -0,0 +1,20 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { BASE_URL } from '@/components/constants/url';
+
+const baseQuery = fetchBaseQuery({
+    baseUrl: BASE_URL
+})
+
+export const api = createApi({
+    baseQuery,
+    /**
```

Comment:
I believe we don't require these comments here.
If I am not wrong, do remove these comments.

Reply:
Comments are meant only when code cannot be understood by reading it.
If someone is new, then must read about RTK>